Makefile

```diff
@@ -35,6 +35,7 @@ IMG_BASE ?= quay.io/sustainable_computing_io
 # OPERATOR_IMG define the image:tag used for the operator
 # You can use it as an arg. (E.g make operator-build OPERATOR_IMG=<some-registry>:<version>)
 OPERATOR_IMG ?= $(IMG_BASE)/kepler-operator:$(VERSION)
+ADDITIONAL_TAGS ?=
```

Comment:
during local development, does `operator-build`, `operator-push` get additional tags?

Reply:
since `ADDITIONAL_TAGS` is empty, it should not create any additional tags. Does this break on your local dev env?

Reply:
did not test it yet.